src/screens/IndexScreen.js

```diff
@@ -394,12 +431,32 @@ export const IndexScreen = ({ navigation, route }) => {
                         <>
                           <DropdownHeader
                             {...{
+                              data:
+                                (initialNewsItemsFetch || initialEventRecordsItemsFetch) && loading
+                                  ? {}
+                                  : data,
                               query,
                               queryVariables,
-                              data: initialNewsItemsFetch && loading ? {} : data,
                               updateListData: updateListDataByDropdown
                             }}
                           />
+
+                          {query === QUERY_TYPES.EVENT_RECORDS && !!eventRecordsAddressesData && (
+                            <DropdownHeader
+                              {...{
+                                data:
+                                  (initialEventRecordsItemsFetch && loading) ||
+                                  eventRecordsAddressesLoading
+                                    ? []
+                                    : eventRecordsAddressesData,
```

Comment:
`eventRecordsAddressesData` is actually an object. `eventRecordsAddressesData.eventRecordsAddresses` is an array.

Therefore, if we give array as default here, we encounter key problem. I will fix this as object.
<img width="752" alt="image" src="https://user-images.githubusercontent.com/11755668/229053681-50a766c9-e203-42c3-9679-22418bb4d266.png">

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/3a54535c361440fd3d017100f8e37b5a30b6037e